package.json

```diff
@@ -12,8 +12,7 @@
     "deploy": "yarn build && cp now.json dist/now.json && now --prod",
     "graphql:codegen": "graphql-codegen --config codegen.yml",
     "test": "echo Tests command not implemented.",
-    "snyk-protect": "snyk protect",
-    "prepare": "yarn run snyk-protect"
+    "snyk-protect": "snyk protect"
```

Comment:
Snyk doesn't seem to be working -- the `prepare` script below was failing with a "failed to load policy" error.

Reply:
While at it, could we please move the dependency of Synk (https://github.com/onflow/flow-playground/blob/master/package.json#L55) to the `devDependency` section?

Reply:
Done! https://github.com/onflow/flow-playground/pull/22/commits/17b5c16d0e25092745e4c43d9bf0f822fa98085b